DigitalLearningSolutions.Web/Styles/trackingSystem/courseSetup.scss

```diff
@@ -2,17 +2,25 @@
 @import "../shared/cardWithButtons";
 @import "../shared/searchableElements/searchableElements";
 
-.copy-course-button{
-    background: none;
-    border: none;
-    padding: 0;
-    color: $nhsuk-link-color;
-    @extend .nhsuk-u-font-size-19;
-    text-decoration: underline;
+.copy-course-button {
+  background: none;
+  border: none;
+  padding: 0;
+  color: $nhsuk-link-color;
+  @extend .nhsuk-u-font-size-19;
+  text-decoration: underline;
 
-    &:hover{
-        color: $nhsuk-link-hover-color;
-        text-decoration: none;
-        cursor: pointer;
-    }
+  &:hover {
+    color: $nhsuk-link-hover-color;
+    text-decoration: none;
+    cursor: pointer;
+  }
+
+  &:focus {

```

Comment:
Is there an NHS class that will do all this for us?

Reply:
Good thinking. There is indeed for the :focus, so I've implemented that instead.